index.rst

```diff
@@ -329,6 +329,7 @@ Misc Components
 
     PCF8574 I/O Expander, components/pcf8574, pcf8574.jpg
     MCP230XX I/O Expander, components/mcp230xx, mcp230xx.svg
+    MCP3008 8-Channel 10-Bit A/D Converter, components/mcp3008, mcp3008.png
```

Comment:
ideally we use a very small (bit size) image on the index and if you want a bigger one on the component page. Please use image optimization sw as tinyjpg.com 
More info is available in the contrib guide on esphome docs.

Reply:
Resized in .5